releasing/README.md

```diff
@@ -376,6 +376,42 @@ project [k8s-staging-kustomize].
 Commit and push your changes. Then create a PR to [k8s.io] to promote
 new images. Assign the PR to @monopole and @Shell32-natsu.
 
+## Update kustomize-in-kubectl
+
+[kubernetes/kubernetes]: https://github.com/kubernetes/kubernetes
+
+To update the version of kustomize shipped with kubectl, first
+fork and clone the [kubernetes/kubernetes] repo.
+
+In the root of the kubernetes repo, run the following commands, modifying
+the version numbers to the newly released ones:
```

Comment:
suggest adding the link to show where to find the "newly released ones" since those libraries are mixed with together kustomize releases. https://github.com/kubernetes-sigs/kustomize/releases

Reply:
done, thank you!